### ISSUE TITLE
Expand this repo by absorbing others related to Chinese Government Misconduct

### ISSUE THREAD

**Issue by Faunsce** [caffeine-overload/bandinchina#108]
As we can tell, the censorship of the people not in china (like the us) is being done through companies under chinas thumb, but this isnt the only thing china has been doing and i think in order to help further strength the arguments in favor of the hong kong people that we should expand this repo to be demonstrative of all the shit chinas doing atm. i found a really good repository that a developer over on the notepad++ repos issues told me about (for those of you who dont know, the dev spoke up about uyghurs and now their repo is flooded with pro and anti-chinese discussion). I already brought this up to the maintainer but i think we should try and absorb this repo as well as others like it to expand this repositories purpose. I do think HK freedom should still be the main obj of this repo however. https://github.com/taibangle/awesome-china ignore the awesome part, its satirical

**Comment by caffeine-overload** [comment]
Oh yeah, there is quite a bit of duplicated effort in awesome-china, since they also have a blacklist. Granted, their list has a wider scope than the one in this repo, but certainly incorporating their entries that fit this scope of the blacklist here can be done. 

I also remember @pinkuribon wanted to create a separate list of HK companies but I've heard no word yet, we can start one here but we'd need the help of people from Hong Kong to populate the list.

I can certainly add a list of reports about human rights abuses in China and that list of police brutality.

I know that Hong Kong is the most talked about issue currently, and is the one that started the conversation and inspired this repo, but I don't want to sideline other issues like Taiwan and Xinjiang, and the encompassing issue of companies bending to CCP bullying. After all, many of the entries in the blacklist are apologies for not listing Taiwan as part of the PRC. 

This is why I have the idea to eventually make a "made in Xinjiang" list, since I heard that some cheap fashion outlets were selling items made with prisoner labour in Xinjiang. If that's the case, those companies must be named.

**Comment by Faunsce** [comment]
Yeah, Upgrading this repo will definetly be hard but i should be able to help more as its not rly coding and just organization after the convention im attending in a couple days (making my outfit final pieces as we speak lol) I've also got like 5 copies of this repo in case it gets knocked down but im sure that wont happen. I think a plan could maybe be outline using the Github projects feature or even just a seperate README detailing your short term and long term goals which should help members be able to better provide the necesarry additions.
